esp/esdllib/esdl_script.cpp

```diff
@@ -29,6 +29,61 @@
 #include <fxpp/FragmentedXmlPullParser.hpp>
 using namespace xpp;
 
+class CEsdlScriptContext : public CInterfaceOf<IEsdlScriptContext>
+{
+public: // ISectionalXmlDocModel
+    virtual IXpathContext* createXpathContext(IXpathContext* parent, const char* section, bool strictParameterDeclaration) override { return docModel->createXpathContext(parent, section, strictParameterDeclaration); }
+    virtual IXpathContext* getCopiedSectionXpathContext(IXpathContext* parent, const char* tgtSection, const char* srcSection, bool strictParameterDeclaration) override { return docModel->getCopiedSectionXpathContext(parent, tgtSection, srcSection, strictParameterDeclaration); }
+    virtual void* queryUserData() override { return this; }
+    virtual void setContent(const char* section, const char* xml) override { docModel->setContent(section, xml); }
+    virtual void appendContent(const char* section, const char* name, const char* xml) override { docModel->appendContent(section, name, xml); }
+    virtual void setContent(const char* section, IPropertyTree* tree) override { docModel->setContent(section, tree); }
+    virtual bool tokenize(const char* str, const char* delimiters, StringBuffer& resultPath) override { return docModel->tokenize(str, delimiters, resultPath); }
+    virtual void setAttribute(const char* section, const char* name, const char* value) override { docModel->setAttribute(section, name, value); }
+    virtual const char* queryAttribute(const char* section, const char* name) override { return docModel->queryAttribute(section, name); }
+    virtual const char* getAttribute(const char* section, const char* name, StringBuffer& s) override { return docModel->getAttribute(section, name, s); }
+    virtual const char* getXPathString(const char* xpath, StringBuffer& s) const override { return docModel->getXPathString(xpath, s); }
+    virtual __int64 getXPathInt64(const char* xpath, __int64 dft = false) const override { return docModel->getXPathInt64(xpath, dft); }
+    virtual bool getXPathBool(const char* xpath, bool dft = false) const override { return docModel->getXPathBool(xpath, dft); }
+    virtual void toXML(StringBuffer& xml, const char* section, bool includeParentNode = false) override { docModel->toXML(xml, section, includeParentNode); }
+    virtual void toXML(StringBuffer& xml) override { docModel->toXML(xml); }
+    virtual IPropertyTree* createPTreeFromSection(const char* section) override { return docModel->createPTreeFromSection(section); }
+    virtual void cleanupTemporaries() override { docModel->cleanupTemporaries(); }
+public: // IEsdlScriptContext
+    virtual IEspContext* queryEspContext() const override { return espCtx; }
+    virtual IEsdlFunctionRegister* queryFunctionRegister() const override { return functionRegister; }
+    virtual void setTraceToStdout(bool val) override
+    {
+        if (val != traceToStdout)
+        {
```

Comment:
No need for the brackets for a single line if body.

Reply:
I kept the braces when I separated it from the larger PR, where it is not currently a one line body.